scripts/shared_conda_setup.sh

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env bash
+
+# This is sourced by pcds_conda and dev_conda
+# It contains the shared lines between those two sourceable scripts
+# See pcds_conda and dev_conda
+
+# Filesystem-specific settings
+# Change these if we change the filesystem again
+PCDS_ROOT="/cds/group/pcds"
+SETUP_ROOT="${PCDS_ROOT}/setup"
+PYPS_ROOT="${PCDS_ROOT}/pyps"
+APPS_ROOT="${PYPS_ROOT}/apps"
+CONDA_ROOT="${PYPS_ROOT}/conda"
+
+# Clear externally set paths that may mess with imports etc.
+unset LD_LIBRARY_PATH
+unset PYTHONPATH
+
+# Pick up EPICS environment variable settings just in case user did not
+source "${SETUP_ROOT}/epics-ca-env.sh"
+
+# BASE_ENV transition points
+FIRST_PCDS_PY38="4.0.0"
+FIRST_PCDS_PY39="4.2.0"
```

Comment:
Wow did we really not bump the major version for a python version change?

Reply:
Yeeep